vividus-plugin-parquet/src/main/resources/spring.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+        http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd"
+    default-lazy-init="true">
+
+    <bean class="org.vividus.bdd.expression.GenerateFileExpressionProcessor"/>
```

Comment:
@valfirst does it makes sense to enable scanning?

Reply:
Please, explain what you mean

Reply:
yes, we have such item in backlog, but it's out of scope this PR

Reply:
@tshkursk https://github.com/vividus-framework/vividus/blob/master/vividus/src/main/resources/org/vividus/spring.xml#L10-L20